int_experian/int_experian_sgjc/cartridge/static/default/js/edqUtils.js

```diff
@@ -342,16 +347,21 @@ function enableButtonDisable(buttonToDisable, buttonStatus) {
 	buttonToDisable.disabled = buttonStatus;
 	if (document.getElementById("form-submit")) { document.getElementById("form-submit").disabled = buttonStatus; }
 }
+/** 
+ * Will manage the access restriction when Phone/Email validation is being used based on the BM configuration.
+ */
 function edqPhoneEmailValidationCallback(blnValidationOption, edqSelectorResponse) {
 	/** TASK:101729 Allow users to continue with invalid phone and email; 
      * based on the Business Manager configuration we can set if we want to prevent the user to go through with an invalid phone. */
 	if (!edqSelectorResponse) { return; }
-	if ((blnValidationOption) && (edqSelectorResponse.hasAttribute("edq-metadata"))) {
+	if ((blnValidationOption) && (edqSelectorResponse.hasAttribute("edq-metadata")) && (pageRestrictValidation)) {
 		var edqMetaDataResponse = JSON.parse(edqSelectorResponse.getAttribute("edq-metadata"));
 		if ((edqMetaDataResponse["Certainty"].toLowerCase() == "verified") || (edqMetaDataResponse["Certainty"].toLowerCase() == "unknown")) {
-			enableButtonDisable(edqCurrentSubmitButtonSelector, false);
+			pageRestrictValidation = true;
+			enableButtonDisable(edqCurrentSubmitButton, false);
```

Comment:
What you have is fine, but I'd recommend a name more declarative, like a `enableButton` and `disableButton` function if you're going this route. Otherwise it would be clearer just to use the built in API for this, i.e. `edqCurrentSubmitButton.removeAttribute("disabled")`

Reply:
I'll have it in mind; we might do it in the code refactoring.